app/setup_pf.py

```diff
@@ -116,6 +116,8 @@ def setup_configs():
         seek_query_form_custom_htmls={'en': '', 'fr': ''},
         published_date=get_timestamp(datetime(2010, 1, 12)),
         updated_date=get_timestamp(datetime(2010, 1, 12)),
+        admin_user_email=get_current_user_email(),
```

Comment:
This method is typically called when you first visit a new PF app and click [Setup datastore] button. In that timing, you typically haven't logged in, so get_current_user_email() would return None.

Having said that, I don't think there's a good default value here. You can just initialize with an empty string here, and let the cron job not to send emails when this config is empty. (You can configure the email address later in the admin page.)